recipe/meta.yaml

```diff
@@ -55,8 +55,8 @@ outputs:
         - make      # [linux]
       host:
         # GPU requirements
-        - cudnn                           # [cuda_compiler_version != "None"]
-        - nccl                            # [cuda_compiler_version != "None"]
+        - cudnn                           # [cuda_compiler_version != "None" and (not win)]
+        - nccl                            # [cuda_compiler_version != "None" and (not win)]
```

Comment:
Why?

Reply:
I'll remove these. This was an artifact before I added a skip to the windows build.